tests/test_audit.py

```diff
@@ -0,0 +1,67 @@
+# from tests import factories
+# from tests.common import ApiBaseTest
+# from webservices.common.models import AuditCase
+# from webservices.resources.audit import AuditCaseView
+# from webservices.schemas import AuditCaseSchema
+# from webservices.rest import api
```

Comment:
Same note as above regarding commented out lines.

Reply:
Disregard previous comment, this is going to be worked on shortly after, we're still debugging through pieces.